sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -66,6 +66,8 @@ struct _pi_platform {
   // a pretty good fit to keep here.
   ze_driver_handle_t ZeDriver;
 
+  ze_context_handle_t ZeContext;
```

Comment:
Didnt find `zeContextCreate()`. I think we are missing it. Best place to put it for now would be right after `zeDriverGet.`

Reply:
Right, added now. But I believe that the PI platform is not the right place for L0 context. Tagging @againull who is looking into this.